packages/asc-ui/src/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,152 @@
+import { ChevronRight, Download } from '@datapunt/asc-assets'
+import { storiesOf } from '@storybook/react'
+import styled from '@datapunt/asc-core'
+import React from 'react'
+import Icon from '../Icon'
+import Image from '../Image'
+import Card from '.'
+import CardActions from './CardActions'
+import CardContent from './CardContent'
+import CardMedia from './CardMedia'
+import Heading from '../Heading'
+import Paragraph from '../Paragraph'
+
+const CardContentStyled = styled(CardContent)`
+  padding-top: 8px;
+  padding-bottom: 8px;
+`
+
+storiesOf('Composed/Card', module)
+  .addDecorator(storyFn => (
+    <div style={{ padding: '40px 10px', background: 'rgb(248,248,248,0.8)' }}>
+      {storyFn()}
+    </div>
+  ))
+  .add('default', () => (
+    <Card maxWidth={250}>
+      <CardContent>
+        <Heading as="h6">This is a card</Heading>
+        <Paragraph>Lorem ipsum dolor sit amet</Paragraph>
+      </CardContent>
+    </Card>
+  ))
+  .add('default with action', () => (
+    <Card maxWidth={250}>
+      <CardContent>
+        <Heading as="h6">This is a card</Heading>
+        <Paragraph>Lorem ipsum dolor sit amet</Paragraph>
+      </CardContent>
+      <CardActions>
+        <Icon size={15}>
+          <ChevronRight />
+        </Icon>
+      </CardActions>
+    </Card>
+  ))
+  .add('default with action and media item', () => (
+    <Card maxWidth={250}>
```

Comment:
The picture has a max-width, perhaps in this case it should not right? should be full width...

Reply:
Good point. I've hanged the max-width to horizontal Cards only